### DISCUSSION TITLE
Trying send email via AWS SES using the Emailjs.com service but get this error, SMTP: Mail command failed: 501 Invalid MAIL FROM address provided

### DISCUSSION THREAD

**Discussion by danieldmu** [emailjs-com/help#133] [Q&A] [answered]
The AWS SES credentials are working fine because I can send test emails using powershell, the EmailJS interface does not have a way to modify the Mail From. 

I tyied diferent SES credentials with no results. But I think having the ability to modify the mail from is key to solve it, because SES expects an especific mail from. Seems to me like the emailjs service is trying to use the username as the mail from. 

I´m attaching the image from emailjs configuration, there is no mail from field. 
![EMAILJS SAMPLE](https://github.com/user-attachments/assets/d9654e17-56ff-4423-857d-bb92e8fea23a)

**Comment by xr0master** [comment]
<img width="589" alt="image" src="https://github.com/user-attachments/assets/03ba58e4-4b51-490d-a4aa-aeb20910ce08">

**Reply by xr0master** [reply, marked as answer]
There is a ready-made provider for SES, but you can use it as you like.
After you create the service, specify the correct "From Email" value in your template.
